src/pages/shop/Shop.tsx

```diff
@@ -1,15 +1,21 @@
-import Banner from '../../components/banner/Banner';
+import Banner from '../../components/Banner/Banner';
+import ProductCard from '../../components/productCard/ProductCard';
 import React from 'react';
 
 export default function Shop() {
   return (
-    <section className="Shop">
-      <Banner
-        title="Shop"
-        pageName="Shop"
-        coverPhoto="https://casona.familab.net/wp-content/uploads/2021/09/heading_bg_dark.jpg"
-        sliderExist={true}
-      />
-    </section>
+    <>
+      <section className="Shop">
+        <Banner
+          title="Shop"
+          pageName="Shop"
+          coverPhoto="https://casona.familab.net/wp-content/uploads/2021/09/heading_bg_dark.jpg"
```

Comment:
I think it's better not to have links to the sourse cite))
you ca just download this image and put it next to tsx)